.travis.yml

```diff
@@ -110,8 +83,7 @@ jobs:
       # do not inherit
       install: []
       before_script:
-        - export DESIRED_VERSION="v3.1.0"
```

Comment:
if this is controlling the version dowloaded by get-helm3, we should move it to line 15, so that the other helm install in line 43 uses the same version

Reply:
or we don't and use the latest helm